tests/bluetooth/tester/src/btp_vcp.c

```diff
@@ -0,0 +1,553 @@
+/* btp_vcp.c - Bluetooth VCP Tester */
+
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/testing.h>
+#include <zephyr/bluetooth/audio/vcp.h>
+#include <zephyr/bluetooth/audio/aics.h>
+#include "zephyr/bluetooth/audio/vocs.h"
+
+#include <app_keys.h>
+
+#include <zephyr/logging/log.h>
+#define LOG_MODULE_NAME bttester_vcp
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include "btp/btp.h"
+
+#define CONTROLLER_INDEX 0
+
+struct bt_vcp_vol_rend_register_param vcp_register_param;
+struct bt_vcp_included included;
+
+static void set_register_params(uint8_t gain_mode);
+uint8_t tester_init_vcp(void);
+
+
+/* Volume Control Service */
+static void vcs_supported_commands(uint8_t *data, uint16_t len)
+{
+	struct net_buf_simple *buf = NET_BUF_SIMPLE(BTP_DATA_MAX_SIZE);
+
+	net_buf_simple_init(buf, 0);
+	net_buf_simple_add_u8(buf, VCS_READ_SUPPORTED_COMMANDS);
+	net_buf_simple_add_u8(buf, VCS_SET_VOL);
+	net_buf_simple_add_u8(buf, VCS_VOL_UP);
+	net_buf_simple_add_u8(buf, VCS_VOL_DOWN);
+	net_buf_simple_add_u8(buf, VCS_MUTE);
+	net_buf_simple_add_u8(buf, VCS_UNMUTE);
+
+	tester_send(BTP_SERVICE_ID_VCS, VCS_READ_SUPPORTED_COMMANDS,
+		    CONTROLLER_INDEX, buf->data, buf->len);
+}
+
+static void set_volume(uint8_t *data, uint16_t len)
+{
+	const struct vcs_set_vol_cmd *cmd = (void *)data;
+	uint8_t volume;
+	int err;
+
+	volume = sys_le16_to_cpu(cmd->volume);
+
+	LOG_DBG("volume 0x%04x", volume);
```

Comment:
Assuming that `cmd->volume` is also a `uint8_t`
```suggestion
	uint8_t volume;
	int err;

	volume = cmd->volume;

	LOG_DBG("volume 0x%02x", volume);
```

Reply:
Done